vm/systemSmartContracts/governance.pb.go

```diff
@@ -389,6 +389,108 @@ func (m *OngoingVotedList) GetDelegated() []uint64 {
 	return nil
 }
 
+type OngoingVotedListV2 struct {
+	Direct               []uint64                `protobuf:"varint,1,rep,packed,name=Direct,proto3" json:"Direct"`
+	Delegated            []uint64                `protobuf:"varint,2,rep,packed,name=Delegated,proto3" json:"Delegated"`
+	DelegatedWithAddress []*DelegatedWithAddress `protobuf:"bytes,3,rep,name=DelegatedWithAddress,proto3" json:"DelegatedWithAddress"`
+}
+
+func (m *OngoingVotedListV2) Reset()      { *m = OngoingVotedListV2{} }
+func (*OngoingVotedListV2) ProtoMessage() {}
+func (*OngoingVotedListV2) Descriptor() ([]byte, []int) {
+	return fileDescriptor_e18a03da5266c714, []int{4}
+}
+func (m *OngoingVotedListV2) XXX_Unmarshal(b []byte) error {
+	return m.Unmarshal(b)
+}
+func (m *OngoingVotedListV2) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	b = b[:cap(b)]
+	n, err := m.MarshalToSizedBuffer(b)
+	if err != nil {
+		return nil, err
+	}
+	return b[:n], nil
+}
+func (m *OngoingVotedListV2) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_OngoingVotedListV2.Merge(m, src)
+}
+func (m *OngoingVotedListV2) XXX_Size() int {
+	return m.Size()
+}
+func (m *OngoingVotedListV2) XXX_DiscardUnknown() {
+	xxx_messageInfo_OngoingVotedListV2.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_OngoingVotedListV2 proto.InternalMessageInfo
+
+func (m *OngoingVotedListV2) GetDirect() []uint64 {
+	if m != nil {
+		return m.Direct
+	}
+	return nil
+}
+
+func (m *OngoingVotedListV2) GetDelegated() []uint64 {
+	if m != nil {
+		return m.Delegated
+	}
+	return nil
+}
+
+func (m *OngoingVotedListV2) GetDelegatedWithAddress() []*DelegatedWithAddress {
+	if m != nil {
+		return m.DelegatedWithAddress
+	}
+	return nil
+}
+
+type DelegatedWithAddress struct {
+	Nonce            uint64 `protobuf:"varint,1,opt,name=Nonce,proto3" json:"Nonce,omitempty"`
+	DelegatedAddress []byte `protobuf:"bytes,2,opt,name=DelegatedAddress,proto3" json:"DelegatedAddress"`
```

Comment:
In a future PR: we need to find a way to clean all the history votes as the state on the metachain will get larger and larger

Reply:
👍 we already have this on the TODO list